src/components/info-tile/examples/info-tile-progress.tsx

```diff
@@ -26,11 +27,14 @@ import { Component, h, State } from '@stencil/core';
 })
 export class InfoTileProgressExample {
     @State()
-    private progressValue = 76;
+    private progress: InfoTileProgress = {
+        value: 76,
+        maxValue: 100,
+        prefix: '↑',
+        suffix: 'sek',
```

Comment:
> @LucyChyzhova why you change my stuff? 😅

it's only one place where I "changed your stuff" @Kiarokh - I put a suffix as `sek`.   I  wouldn't say "changed" since there was no suffix 😁, only the value. And because the suffix isn't optional I had to add it.  Yes, I didn't pay attention that it was `EUR` at the `info-tile` component, I was trying to make it work. 

I added a fixup.

Also, I have checked the history but couldn't find more changes to the props.

Reply:
This has been changed, so I'll resolve this thread.